pkg/crypto/pseudo.go

```diff
@@ -108,7 +109,7 @@ func ClientPseudo(nodes []t.NodeID, clients []t.ClientID, ownID t.ClientID, seed
 	if c == nil {
 		if err == nil {
 			// Own ID was not found and Crypto module instantiation was not even attempted.
-			return nil, fmt.Errorf("ownID (%d) not found among clients", ownID)
+			return nil, fmt.Errorf("ownID (%s) not found among clients", ownID)
```

Comment:
Let's use `%v` so we don't need to change this again in the unlikely case we use a different type for IDs in the future (or in a variant of Mir)

Reply:
ok